src/bp-core/bp-core-functions.php

```diff
@@ -1316,7 +1316,7 @@ function bp_core_time_since( $older_date, $newer_date = false ) {
 			}
 
 			// No output, so happened right now.
-			if ( ! (int) trim( $output ) ) {
+			if ( ! (int) $count && ! (int) $count2 ) {
```

Comment:
@shohelbuddyboss `$count` and `$count2` will give undefined notices in the log. In addition, this is not the solution, solution is declaring `$output = ''` so that it doesn't give empty variable notice and go in the condition to assign `$right_now_text` to `$output`.

Reply:
I have fixed the issue now.